repository/t.json

```diff
@@ -3761,6 +3761,18 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "Typst",
+			"details": "https://github.com/hyrious/typst-syntax-highlight",
+			"author": "hyrious",
```

Comment:
You don't need to provide the author here, package control will pick up on your username automatically.

Reply:
Removed.